atm.js

```diff
@@ -1,3 +1,98 @@
-$(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+//adds zero class
+function addRedBalance(selector){
+    if(parseFloat($(selector).html().split('$')[1])==0){
+        $(selector).addClass("zero")
+    }
+}
+//removes zero class
+function removeRedBalance(selector){
+    if(parseFloat($(selector).html().split('$')[1])!==0){
+        $(selector).removeClass('zero')
+    }
+}
+//Doc is loaded
+$(document).ready(function() {
+    //implements the zero class on start
+    addRedBalance(".balance");
+    //pre enter from reloading page with form wrapper
+    $('form').submit(function(event) {
+        event.preventDefault();
+    });
+
+    //deposit function checking
+    $(".deposit:first").on('click', function(event) {
+        event.preventDefault();
+        var depositAmount = parseFloat($('.input:first').val()).toFixed(2)
+        if(!isNaN(depositAmount)){
+            var currentBalance = parseFloat($('.balance:first').html().split('$')[1])
+            $('.balance:first').html('$' + (parseFloat(depositAmount)+currentBalance).toFixed(2));
+            addRedBalance(".balance:first");
```

Comment:
So the way you're choosing your selectors here isn't wrong, but I'd steer on the side of making it as simple and readable as possible. something like $(#checking .balance) is easy for other developers to read..the way you have it, you gotta think about a little. It's cool to learn how to use it, but in actuality that's one most of us would have to pause at.